src/components/KeyboardToolbar/index.tsx

```diff
@@ -57,7 +60,7 @@ const TEST_ID_KEYBOARD_TOOLBAR_NEXT = `${TEST_ID_KEYBOARD_TOOLBAR}.next`;
 const TEST_ID_KEYBOARD_TOOLBAR_CONTENT = `${TEST_ID_KEYBOARD_TOOLBAR}.content`;
 const TEST_ID_KEYBOARD_TOOLBAR_DONE = `${TEST_ID_KEYBOARD_TOOLBAR}.done`;
 
-const KEYBOARD_TOOLBAR_HEIGHT = 42;
+export const KEYBOARD_TOOLBAR_HEIGHT = 42;
```

Comment:
Are we sure we want to export that? 😅 

I created it as an internal variable and didn't suppose it should be exported. The thing that I'm afraid - if in future the size will be dynamic (based on accessibility preferences, for example), then the height will not be a constant anymore, so I think that deriving any layout information via `onLayout` callback is a more preferrable approach at the moment.

What do you think? 👀

Reply:
Yup it might be creating a breaking change for something that can be retrieved by the onLayout. I'll remove the commit

Reply:
Yes, I wasn't sure either. I'll remove the commit